analyzer/src/main/kotlin/managers/SpdxDocumentFile.kt

```diff
@@ -58,6 +58,40 @@ private val SPDX_SCOPE_RELATIONSHIPS = listOf(
     SpdxRelationship.Type.TEST_DEPENDENCY_OF
 )
 
+/**
+ * Return the organization from an "originator", "supplier", or "annotator" string, or null if no organization is
+ * specified.
+ */
+private fun String.extractOrganization(): String? =
```

Comment:
Is there a reason why you explicitly added the return type?

Reply:
Just for consistency with the other functions.